api/notd/manager.py

```diff
@@ -281,7 +283,17 @@ async def retreive_token_metadata(self, registryAddress: str, tokenId: str) -> R
             ], limit=1,
         )
         if len(tokenMetadatas) == 0:
-            raise NotFoundException()
+            tokenMetadatas = []
+            cacheKey = f'{registryAddress}:{tokenId}'
+            if cacheKey in self._tokenCache:
+                return self._tokenCache[cacheKey]
+            try:
+                registryToken = await self.openseaClient.retreive_registry_token(registryAddress=registryAddress, tokenId=tokenId)
+            except NotFoundException:
+                registryToken = await self.tokenClient.retreive_registry_token(registryAddress=registryAddress, tokenId=tokenId)
```

Comment:
no need for this - its the same thing our database has

Reply:
So for this should I remove the try block entirely?

Reply:
yep